experimental/PropertyDDS/packages/property-dds/src/test/createDerivedGuid.ts

```diff
@@ -0,0 +1,17 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import * as crypto from "crypto";
+
+export function createDerivedGuid(referenceGuid: string, identifier: string) {
```

Comment:
This seems like an implementation of uuidv5. MAybe just use https://www.npmjs.com/package/uuid#uuidv5name-namespace-buffer-offset

Reply:
if not using an existing uuidv5, it would be good to document this function as either being a standards compliant uuidv5 or not (and why)

Reply:
Done.

Reply:
if that was done, it seems like it was undone?